hooks/pre-push

```diff
@@ -30,7 +30,11 @@ do
     pushd "${dir}" > /dev/null
     if [ -n "`echo ${files} | grep ^${dir}\/.*\.ts$`" ]; then
       echo "Linting ${dir}..."
-      ./project.rb lint
+      if [ -x "$(command -v yarn)" ]; then
+        yarn run lint
```

Comment:
Out of curiosity, what difference in runtime did you see from this?

Reply:
Everything is to enable running without docker. I'm working on swagger gen happening without docker, too.